agent/structs/config_entry_cluster_oss.go

```diff
@@ -0,0 +1,5 @@
+package structs
+
+func (e *ClusterConfigEntry) validateEnterpriseMeta() error {
```

Comment:
The enterprise version of this method will be lifted from the current version of `ProxyConfigEntry`